### PR TITLE
Time only mode

### PR DESCRIPTION
**Pull request by s-lee-kwong**
Most of the work should be attributed to @a2exfr but I decided to take what they did and try to get it integrated into the project officially.

### CODE REVIEW

**Comment by mariomka** [comment]
I will review soon. Thank you.

**Comment by mylmz10** [comment]
@mariomka Did you started review?

**Comment by xtrasmal** [comment]
is there a status on this request? @mariomka

File: src/DatetimePopup.vue [hunk 1]
```diff
@@ -3,6 +3,8 @@
     <div class="vdatetime-popup__header">
       <div class="vdatetime-popup__year" @click="showYear">{{ year }}</div>
       <div class="vdatetime-popup__date">{{ dateFormatted }}</div>
+      <div class="vdatetime-popup__year" @click="showYear" v-if="type !== 'time'">{{ year }}</div>
+      <div class="vdatetime-popup__date" v-if="type !== 'time'">{{ dateFormatted }}</div>
```
Comment (mariomka) [inline src/DatetimePopup.vue:5]: Something is wrong here. Headers are duplicated.
What is your idea for the header? Show an empty header?

File: test/specs/Datetime.spec.js [hunk 2]
```diff
@@ -265,9 +265,87 @@ describe('Datetime.vue', function () {
...
+          data () {
+            return {
+              datetime: '2017-12-05T00:00:00.000Z'
+            }
+          }
+        })
+
+      expect(vm.datetime).to.be.equal('2017-12-04T19:00:00.000-05:00')
+    })
+
```
Comment (mariomka) [inline test/specs/Datetime.spec.js:344]: Why are you using many  `setTimeout()`? I think they are not necessary.

The first test could be:

```js
const vm = createVM(this,
        `<Datetime v-model="datetime" type='time' zone='UTC-03:00'></Datetime>`,
        {
          components: { Datetime },
          data () {
            return {
              datetime: '2017-12-07T09:00:00.000Z'
            }
          }
        })

      vm.$nextTick(() => {
        expect(vm.$('.vdatetime-input').value).to.be.equal('06:00')
        done()
      })
```

**Review by mariomka — CHANGES_REQUESTED** [review]
Thank you for the PR but please check out the comments.

Sorry for taking a long time to review, I've had a lot of work.

File: src/DatetimePopup.vue (hunk 1 above)
Comment (a2exfr) [reply src/DatetimePopup.vue:5]: I think its by mistake, not removed. Should be [just](https://github.com/a2exfr/vue-datetime/blob/62460e5a69a3a30c30c8fabb19f616a7eedba0e7/src/DatetimePopup.vue#L3-L5)
The idea was  to show small colored  header line. With [this](https://github.com/a2exfr/vue-datetime/blob/v1.x/src/DatetimePopup.vue#L264).

Comment (s-lee-kwong) [reply src/DatetimePopup.vue:5]: I am a bit busy this weekend but I will correct this on monday.

File: test/specs/Datetime.spec.js (hunk 2 above)
Comment (s-lee-kwong) [reply test/specs/Datetime.spec.js:344]: ok i will take a look. thanks

Comment (s-lee-kwong) [reply test/specs/Datetime.spec.js:344]: ok i have fixed what you asked

File: src/DatetimePopup.vue (hunk 1 above)
Comment (s-lee-kwong) [reply src/DatetimePopup.vue:5]: actually i was able to quickly get that fixed.

**Comment by mariomka** [comment]
Merged but I have added the ability to set custom format.

I think the header is a bit strange... any idea?

**Pull request merged**